src/peft/tuners/lora/config.py

```diff
@@ -360,6 +474,15 @@ class LoraConfig(PeftConfig):
             )
         },
     )
+    corda_config: Union[CordaConfig, dict] = field(
```

Comment:
I think it's clearer if we make this argument `Optional`  and initialize it as `None` by default (below we already check for `None`).